dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -55,6 +55,20 @@ def submit_evaluations
     end
   end
 
+  # GET /rubrics/:id/get_ai_evaluations
+  def get_ai_evaluations
+    return head :forbidden unless current_user&.teacher?
+
+    permitted_params = params.transform_keys(&:underscore).permit(:id, :student_id)
+    student = User.find(permitted_params[:student_id])
+    return head :not_found unless student
+    return head :forbidden unless can?(:manage, student)
+
+    learning_goals = LearningGoal.where(rubric_id: permitted_params[:id])
+    learning_goal_ai_evaluations = LearningGoalAiEvaluation.where(user_id: permitted_params[:student_id], learning_goal_id: learning_goals.pluck(:id))
```

Comment:
would we also want to filter to just the most recent evaluation for each learning goal? this is what I imagine we'll need to populate the new rubric UI, but I could be missing how this is going to be used exactly

Reply:
oh good call!